### PR TITLE
Edited readme and added new logo image

### PR DESCRIPTION
**Pull request by markaleksanderh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
+</h1>
 
-## Private, secure, decentralized
+<p align="center">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/downloads/simplex-chat/simplex-chat/total">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/v/release/simplex-chat/simplex-chat">
+  <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/SimpleXChat?style=social">
+</p>
 
-[![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
```
Comment (efim-poberezkin) [inline README.md:7]: The new header looks cool!

Regarding badges, I think we should keep this one

File: README.md [hunk 2]
```diff
@@ -1,44 +1,51 @@
...
+<h1 align="center">
+  <img src="images/simplex-chat-logo.svg" alt="SimpleX logo">
 
-# SimpleX chat
+</h1>
 
-## Private, secure, decentralized
+<p align="center">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/downloads/simplex-chat/simplex-chat/total">
+  <img alt="GitHub all releases" src="https://img.shields.io/github/v/release/simplex-chat/simplex-chat">
```
Comment (efim-poberezkin) [inline README.md:8]: fix alt here?
```suggestion
  <img alt="GitHub release" src="https://img.shields.io/github/v/release/simplex-chat/simplex-chat">
```

File: README.md [hunk 3]
```diff
@@ -1,44 +1,51 @@
...
+Once the chat client is installed, simply run `simplex-chat` from your terminal.
 
-### Welcome word
+### :wave: Welcome
 
-**We are building the most private and secure chat in the world** - open-source, decentralized, and without global identities of any kind. If you would like to support it, you can do so in the following ways:
+**We are building the world's most private and secure chat**. If you would like to support it, you can do so in the following ways:
 
-- 🌟 **Star it on GitHub** - it's a great feedback you like it and helps us raise the visibility of the project.
+- 🌟 **Star it on GitHub** - starring the repo helps us raise the visibility of the project.
```
Comment (efim-poberezkin) [inline README.md:32]: my grammar sense may be off here, but I read it as "*us* starring the repo [as in activity done by us] helps ....", maybe adjust the wording here? Not sure

File: README.md [hunk 4]
```diff
@@ -58,29 +65,35 @@ We appreciate all the help from our contributors, thank you!
...
+Unlike federated networks, the participating server nodes **do not have records of the users**, **do not communicate with each other**, **do not store messages** after they are delivered to the recipients, and there is no way to discover the full list of participating servers. SimpleX network avoids the problem of metadata visibility that federated networks suffer from and better protects the network, as servers do not communicate with each other. Each server node provides unidirectional "dumb pipes" to the users, that do authorization without authentication, having no knowledge of the the users or their contacts. Each queue is assigned two RSA keys - one for receiver and one for sender - and each queue access is authorized with a signature created using a respective key's private counterpart.
```
Comment (efim-poberezkin) [inline README.md:89]: make this bold as well? Also not sure, just feels strange when it's the only one left plain
```suggestion
Unlike federated networks, the participating server nodes **do not have records of the users**, **do not communicate with each other**, **do not store messages** after they are delivered to the recipients, and **there is no way to discover the full list of participating servers**. SimpleX network avoids the problem of metadata visibility that federated networks suffer from and better protects the network, as servers do not communicate with each other. Each server node provides unidirectional "dumb pipes" to the users, that do authorization without authentication, having no knowledge of the the users or their contacts. Each queue is assigned two RSA keys - one for receiver and one for sender - and each queue access is authorized with a signature created using a respective key's private counterpart.
```

File: README.md (hunk 1 above)
Comment (markaleksanderh) [reply README.md:7]: Good suggestion, I missed this one off. The Git badges don't render properly so I'd need to find the Shields equivalent to get this to render correctly

File: README.md (hunk 3 above)
Comment (markaleksanderh) [reply README.md:32]: Fair point - _starring the repo helps raise the visibility of the project_ sounds better

**Pull request merged**